vtest/gen-ref.bat

```diff
@@ -1,7 +1,7 @@
 
 rem create reference
 
-set MSCORE=..\msvc.install_x64\bin\musescore.exe
+set MSCORE=..\msvc.install_x64\bin\musescore3.exe
 set DPI=130
 
 %MSCORE% %1.mscz -r %DPI% -o %1.png
```

Comment:
This however still needs to get amended, mscx rather then mscz. Sows how very old that file is...